example/src/App.tsx

```diff
@@ -77,28 +83,33 @@ const HomeScreen = ({ navigation }: NativeStackScreenProps<any, any, any>) => {
 const Stack = createNativeStackNavigator();
 const App = () => {
   return (
-    <NavigationContainer>
-      <Stack.Navigator
-        screenOptions={({ route }) => {
-          return {
-            headerShown: route.name === 'NavigationHeader',
-          };
-        }}
-      >
-        <Stack.Screen name="Examples" component={HomeScreen} />
-        {examples.map((example) => (
-          <Stack.Screen
-            key={example.name}
-            name={example.name}
-            component={example.component}
-          />
-        ))}
-      </Stack.Navigator>
-    </NavigationContainer>
+    <GestureHandlerRootView style={homeStyles.root}>
```

Comment:
why needed? (maybe a dumb question)

Reply:
This is the primary setup requirement to ensure [react-native-gesture-handler](https://docs.swmansion.com/react-native-gesture-handler/docs/) works in the app (which is the dependency of [@gorhom/bottom-sheet](https://ui.gorhom.dev/components/bottom-sheet/))